### PR TITLE
PHPC-1173: Reimplement replica set seedlist tests

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

File: tests/connect/replicaset-seedlist-001.phpt [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
 <?php skip_if_not_replica_set(); ?>
 <?php skip_if_not_clean(); ?>
-<?php PREDICTABLE(); ?>
 --FILE--
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
+$secondary = get_secondary_server(URI);
```
[the remark this answers is not in the document]
Comment (jmikola) [reply tests/connect/replicaset-seedlist-001.phpt]: I'm fine without if you'd like to test this locally. It's something we should certainly intend to test on Evergreen, but we should be fairly conservative in our use of Travis resources :)

File: tests/utils/skipif.php [hunk 2]
```diff
@@ -39,6 +39,86 @@ function skip_if_not_replica_set()
...
 
+/**
+ * Skips the test if the topology has no arbiter.
+ */
+function skip_if_no_arbiter()
+{
+    try {
+        $primary = get_primary_server(URI);
+    } catch (ConnectionException $e) {
+        exit('skip server is not accessible: ' . $e->getMessage());
```
Comment (jmikola) [inline tests/utils/skipif.php]: Should this message also say "primary server" for consistency?

File: tests/connect/replicaset-seedlist-001.phpt (hunk 1 above)
[the remark this answers is not in the document]
Comment (derickr) [reply tests/connect/replicaset-seedlist-001.phpt]: I added one anyway...

File: tests/utils/skipif.php (hunk 2 above)
Comment (derickr) [reply tests/utils/skipif.php]: Will fix before merging.

**Pull request merged**